src/Sentry.Unity.Editor/SentryWindow.cs

```diff
@@ -90,9 +123,16 @@ private void ValidateDsn()
         public void OnLostFocus()
         {
             Validate();
-            AssetDatabase.SaveAssets();
-            // TODO: This should be gone
+
+            // Saving `UnitySentryOptions` and not `UnitySentryOptionsJson` for OnGUI backward-compat

```

Comment:
We don't need to be backwards compatible here since there hasn't been a single release.

Reply:
`quick-dirty-example-of-json-as-config`